packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -1461,22 +1681,38 @@ Content-Type: application/json;charset=UTF-8
 
 #### Rule properties
 
-| Property     | Description                                                                                  | Type                                    | Required for create   | Required for update |
-| :----------- | :------------------------------------------------------------------------------------------- | :-------------------------------------- | :-------------------- | :------------------ |
-| actions      | An object that contains the `tokens` array, which shows lifetime durations for the tokens    | Object                                  | System generated      | System generated    |
-| conditions   | Specifies the people, Groups, grant types, and Scopes that the rule is applied to             | [Condition object](#condition-object)   | False                 | False               |
-| id           | ID of the rule                                                                               | String                                  | False                 | True                |
-| name         | Name of the rule                                                                             | String                                  | True                  | True                |
-| status       | Specifies whether requests have access to this Claim. Valid values: `ACTIVE` or `INACTIVE`   | Enum                                    | True                  | True                |
-| system       | Specifies whether the rule was created by Okta                                        | Boolean                                 | True                  | True                |
+| Property      | Description                                                        | Data Type                                      | Required for Create  | Required for update    |
```

Comment:
nit - create instead of Create

Reply:
I think this would still be "C" - Create is a CRUD operation - it's a thing. So, let's keep it capitalized.